sci-libs/pg2plplot/pg2plplot-5.13.0.ebuild

```diff
@@ -14,19 +16,20 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="X png postscript static-libs"
 
-DEPEND="sci-libs/plplot[fortran]"
+DEPEND="virtual/fortran
+		sci-libs/plplot[fortran]"
 
 # If USE="png" or "postscript", ensure PLplot has USE="cairo":
 RDEPEND="${DEPEND}
-	 sci-libs/plplot[fortran,X?]
-	 png? ( sci-libs/plplot[cairo] )
-	 postscript? ( sci-libs/plplot[cairo] )"
+		 sci-libs/plplot[fortran,X?]
+		 png? ( sci-libs/plplot[cairo] )
+		 postscript? ( sci-libs/plplot[cairo] )"
 
 src_configure() {
 	local mycmakeargs=(
-		$(cmake-utils_use static-libs CREATE_STATICLIB)
+		-DCREATE_STATICLIB="$(usex static-libs)"
```

Comment:
Can you just remove the `static-libs` option?
Unless specifically needed by another package, it is recommended to disable the static-libs option.

Users can always enable it locally by using `CMAKE_EXTRA_CACHE_FILE` for specifying `static-libs`.

Reply:
I can, but I'm not sure how to build the static libs then.  I would normally test this using `USE=static-libs ebuild pg2plplot-5.13.0.ebuild clean manifest install`.  How can I test it now?  I've googled on replacing the Gentoo `USE` flags with `CMAKE_EXTRA_CACHE_FILE` but I get very few hits.

Reply:
An example for creating a cmake cache file which will enable `BUILD_STATIC_LIBS` for any **<category/package>**.

- Create a folder `/etc/portage/cache`
- Create a file `/etc/portage/cache/CMAKE_STATIC` with the contents
```
set(BUILD_STATIC_LIBS ON CACHE BOOL "")
```
- Create a file `/etc/portage/env/cmake-static` with the contents
```
CMAKE_EXTRA_CACHE_FILE=/etc/portage/cache/CMAKE_STATIC
```
- Create a file `/etc/portage/package.env/<package>` with the contents
```
<category/package>    cmake-static
```

This will enable this option for **<category/package>**.

To enable this for any package, just do the `/etc/portage/package.env/<package>` step for it.   
In fact, using such a cache file, you can override all default arguments set in the ebuild.